Assets/UI/Scenes/HelpUI.tscn

```diff
@@ -0,0 +1,50 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://Assets/UI/Scripts/HelpUI.gd" type="Script" id=1]
+[ext_resource path="res://Assets/UI/Themes/MenuTheme.tres" type="Theme" id=2]
+
+[node name="Node2D" type="Control"]
```

Comment:
A Control with the name Node2D? The name should be changed to "Help" or "HelpUI" or something.

Reply:
Of course. Changed.